packages/next/project.json

```diff
@@ -51,6 +51,11 @@
             "glob": "**/*.d.ts",
             "output": "/"
           },
+          {
+            "input": "packages/next",
+            "glob": "plugins/**/*.html",
+            "output": "/"
+          },
```

Comment:
I'm sure there's a reason, but why is this added? I don't see any changes that would make this needed now.

Reply:
Ah forgot to remove that when removing the next js support. Thanks for catching it!